content/en/docs/reference/commands/istioctl/index.html

```diff
@@ -3111,7 +3111,7 @@ <h2 id="istioctl-manifest-generate">istioctl manifest generate</h2>
 <td><code>-s</code></td>
 <td>Override an IstioOperator value, e.g. to choose a profile
 (--set profile=demo), enable or disable components (--set components.policy.enabled=true), or override Istio
-settings (--set values.global.mtls.enabled=true). See documentation for more info:
+settings (--set values.global.controlPlaneSecurityEnabled=true). See documentation for more info:
```

Comment:
Does the change on this line mean that global.mtls.enabled has the same semantic as global.controlPlaneSecurityEnabled?

Reply:
No. The example is just to show overring default value, so it doesn't matter.

Reply:
We are also getting rid of controlPlaneSecurityEnabled so we might want a different one here as well :slightly_smiling_face:

Reply:
Ok :). Changed to `--set values.grafana.enabled=true`

Reply:
I believe this file gets autogenerated (see the note at the top). Any changes here will likely be overwritten when the next merge is done. I believe you need to update this in the istio/istio repo. @snible can help identify the file(s) if a simple search doesn't find them.

Reply:
Ah, thanks. Revert this change and create https://github.com/istio/istio/pull/21645